iocore/net/NetVConnection.cc

```diff
@@ -43,3 +44,39 @@ NetVConnection::cancel_OOB()
 {
   return;
 }
+
+const char *
+NetVCOptions::get_proto_string() const
+{
+  const char *retval;
+  switch (ip_proto) {
+  case USE_TCP:
+    retval = TS_PROTO_TAG_TCP;
+    break;
+  case USE_UDP:
+    retval = TS_PROTO_TAG_UDP;
+    break;
+  default:
+    retval = NULL;
+    break;
+  }
+  return retval;
+}
```

Comment:
I think this would be more succinct:
```C
switch (ip_proto) {
  case USE_TCP:
    return TS_PROTO_TAG_TCP;
 case USE_UDP:
  return TS_PROTO_TAG_UDP;
 default:
    return NULL;
 }
```